src/config.py

```diff
@@ -33,6 +33,14 @@ def __copy__(self):
     def get(self, key, default=None):
         return self.__getitem__(key) if self.__contains__(key) else default
 
+    def pop(self, key, no_exception=False):
+        try:
+            return self.params.pop(key)
```

Comment:
dict.pop() can take a second arg as default value to return if key is not exists in dictionary